test/test_encryption.py

```diff
@@ -1942,6 +1942,32 @@ def test_via_loading_shared_library(self):
         with self.assertRaises(ServerSelectionTimeoutError):
             no_mongocryptd_client.db.command("ping")
 
+    # https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#20-bypass-creating-mongocryptd-client-when-shared-library-is-loaded
+    @unittest.skipUnless(os.environ.get("TEST_CRYPT_SHARED"), "crypt_shared lib is not installed")
+    def test_client_via_loading_shared_library(self):
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
```

Comment:
This test is missing the `listenerThread` portion of the linked spec.

Reply:
Yup, working on doing that right now.